pkg/virt-controller/watch/pool.go

```diff
@@ -969,6 +977,8 @@ func (c *PoolController) execute(key string) error {
 			// Handle updates after scale operations are satisfied.
 			syncErr = c.update(pool, vms)
 		}
+
+		defer virtControllerPoolWorkQueueTracer.StepTrace(key, "sync", trace.Field{Key: "VMPool Name", Value: pool.Name})
```

Comment:
Since `sync` isn't a function here, shouldn't this look like this: https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/watch/vm.go#L320 ?  Otherwise, the 'sync' step is running after sync, updateStatus, and execute.  Also, `updateStatus` could use a trace.

Reply:
ha, this was a mistake. I need to remove the defer. copy paste error :) 

I removed the `defer` here and added a trace to update status